src/renderer/components/KeyboardShortcutHint.tsx

```diff
@@ -0,0 +1,250 @@
+import React, { useEffect, useState } from 'react'
+import { DesktopSettingsType } from '../../shared/shared-types'
+import { runtime } from '../runtime'
+
+const keySymbols: { [key: string]: string } = {
+  Control: '^',
+  Shift: '⇧',
+  Alt: '⌥',
+  Meta: '⌘',
+  Enter: '↵',
+  ArrowDown: '↓',
+  ArrowUp: '↑',
+}
```

Comment:
Maybe rewrite to const KEY_SYMBOL_{CONTROL;SHIFT;ALT...} = '^' instead of a hashmap?